harness/determined/common/api/__init__.py

```diff
@@ -1,4 +1,4 @@
-from determined.common.api import authentication, errors, metric, request
+from determined.common.api import analytics, authentication, errors, metric, request
```

Comment:
I think we should not be importing this in the top-layer namespace at all.  This would basically cause it to show up in the user's IDE when they try to tab complete after `determined.<TAB>`.

Removing it here will probably not affect any of your other code since you're following the google import guide anyway.

Reply:
will delete it